File: ESAResturant.NEW.web/ClientApp/src/app/admin/bill-invoice/Models/BillDeliveries.ts
import { Bill } from "./Bill";

export class BillDeliveries  {

  constructor() {

    this.id = 0;
    this.billId = 0;
    this.deliveryName = '';
    this.deliveryPhone = '';
    this.deliveryAddress = '';
  }
  
  id: number;
  deliveryName: string;
  deliveryPhone: string;
  deliveryAddress: string;
  billId: number;
  bill: Bill;
}

File: ESC.Resturant.Data/Entities/BillDeliveries.cs
using ESC.Resturant.Data.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public class BillDeliveries
    {
        [Key]
        public int Id { get; set; }
        public Nullable<int> BillId { get; set; }
        public string DeliveryName { get; set; }
        public string DeliveryPhone { get; set; }
        public string DeliveryAddress { get; set; }
        public virtual Bill Bill { get; set; }
    }
}
